src/server.js

```diff
@@ -202,6 +210,48 @@ app.get(
   }
 );
 
+app.get(
+  `${PREFIX_API}/:${URL_PARAM_EVENT_ID}/restaurants`,
+  getEvent,
+  async (request, response) => {
+    const { event } = request;
+    const users = event.users || {};
+    const userInput = Object.values(users);
+
+    if (userInput.length > 0) {
+      const lat = userInput[0].lat.toString();
+      const long = userInput[0].long.toString();
+      let radius = "50000";
+      let type = "restaurant";
+      let minprice = "0";
+      let maxprice = "4";
+      const restarantData = await (
+        await fetch(
+          `https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=${lat},${long}&radius=${radius}&type=${type}&minprice=${minprice}&maxprice=${maxprice}&key=${env.API_KEY}`
+        )
+      ).json();
+      try {
```

Comment:
Why is the `response.json` in the try block? Is the `fetch` meant to also be in the try block? Also, the fetch request can still fail without throwing an exception. So it's important to check for `response.ok`, or whatever the variable is inside the returned object, to see if the request actually went through.

Reply:
I am marking this comment as unresolved because I think it would be helpful to check the api response status as well. Event if no error is thrown, the response status can come back with some kind of error.